Components/GameBoard/Card.jsx

```diff
@@ -166,21 +166,20 @@ class InnerCard extends React.Component {
             return null;
         }
 
-        var facedownDupes = this.props.card.dupes.filter(card => {
-            return card.facedown;
-        });
-
-        if(!facedownDupes || facedownDupes.length === 0) {
-            return;
-        }
+        let index = 1;
+        let dupes = this.props.card.dupes.map(dupe => {
+            if(this.props.card.facedown) {
+                dupe.facedown = true;
```

Comment:
It might be useful to add a comment here to call out this is explicitly a fix for dragged dupes during setup phase appearing face up. I barely remembered the context of what this was doing today and we talked about it only a day or so ago.